models/user.js

```diff
@@ -21,13 +23,25 @@ const Role = {
 
 const User = new Schema({
   facebookId: String,
-  name: { type: String, required: true, minlength: 2 },
-  surname: { type: String, required: true, minlength: 2 },
+  name: { type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 100
+  },
+  surname: { type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 100
+  },
```

Comment:
Man who made stackoverflow has really interesting blog and among all other niceties he assembled list of new programming jargon: https://blog.codinghorror.com/new-programming-jargon/
Note **3. Egyptian Brackets**. It is used to describe following indent scheme:
```c
if (a == b) {
  printf("hello");
}
```
However you just invented _new direction_: **Egyptian objects**
```js
name: { type: String,
  required: true,
  minlength: 2,
  maxlength: 100
},
```